app/views/katello/api/v2/flatpak_remote_repositories/show.json.rabl

```diff
@@ -0,0 +1,13 @@
+object @resource
+
+extends "katello/api/v2/flatpak_remote_repositories/base"
+
+child :flatpak_remote => :flatpak_remote do
+  attributes :id, :name, :url
+end
+
+if ::Foreman::Cast.to_bool params[:manifests]
```

Comment:
Does there need to be an apipie doc string for this?

Reply:
Added api doc string to controller for this.